### PR TITLE
Fix/code styling

### PR DESCRIPTION
**Pull request by svssathvik7**
- fixed and updated code block styling issues
- fix build time css breakdowns
- reword descriptions with sentence casing

### CODE REVIEW

File: static/swagger/openapi.json [hunk 1]
```diff
@@ -33,12 +33,12 @@
     {
       "name": "relayer",
       "x-displayName": "Gasless API",
-      "description": "The Gasless API enables secure initiation, redemption, and intent creation for cross-chain swaps with health monitoring."
+      "description": "The gasless API enables secure initiation, redemption, and intent creation for cross-chain swaps with health monitoring."
```
Comment (Gowtham118) [inline static/swagger/openapi.json:36]: Change Gasless API to Gasless.

Comment (svssathvik7) [reply static/swagger/openapi.json:36]: Okay

Comment (svssathvik7) [reply static/swagger/openapi.json]: I have pushed the reword commit

File: static/swagger/openapi.json [hunk 2]
```diff
@@ -33,12 +33,12 @@
...
     },
     {
       "name": "auctionhouse",
       "x-displayName": "Auction House",
-      "description": "At the core of the Garden lies the auctionhouse. It is responsible for intent creation, matching, and settlement. One can create intents, track their status, and securely settle them using Atomic Swaps."
+      "description": "At the core of the garden lies the auctionhouse. It is responsible for intent creation, matching, and settlement. One can create intents, track their status, and securely settle them using atomic swaps."
```
Comment (Gowtham118) [inline static/swagger/openapi.json:41]: garden can be Garden, for uniformity in brand

**Review by Gowtham118 — APPROVED** [review]

**Pull request merged**